libcontainer/specconv/spec_linux.go

```diff
@@ -184,6 +202,30 @@ func KnownMountOptions() []string {
 	return res
 }
 
+// KnownMemoryPolicyModes returns the list of the known memory policy modes.
+// Used by `runc features`.
+func KnownMemoryPolicyModes() []string {
+	initMaps()
+	var res []string
+	for k := range mpolModeMap {
+		res = append(res, string(k))
+	}
+	sort.Strings(res)
+	return res
```

Comment:
nit: Does `slices.Sorted(maps.Keys(mpolModeMap))` not work here because of types? Can you make the map `map[string]...` to make the types work so we don't need to open code this?

Reply:
Switched to `map[string]...` that was needed to use `slices.Sorted(...)`.